lms/djangoapps/certificates/management/commands/fix_ungraded_certs.py

```diff
@@ -52,9 +52,9 @@ def handle(self, *args, **options):
         ).filter(grade__exact='')
         course = courses.get_course_by_id(course_id)
         for cert in ungraded:
-            if is_using_certificate_allowlist_and_is_on_allowlist(cert.user, course_id):
-                log.info(f'{course_id} is using allowlist certificates, and the user {cert.user.id} is on its '
-                         f'allowlist. Certificate will not be regraded')
+            if can_generate_certificate_task(cert.user, course_id):
```

Comment:
I can understand that we wouldn't want to regrade the certificate of a learner who is on the allowlist. Wouldn't we want to regrade learners with "regular" v2 certificates though?

Reply:
In the current design for V2, we only set a grade in a cert at the same time that we set the cert status (to keep the cert consistent). This command only sets the grade.

Reply:
Thanks for the clarification!